### DISCUSSION TITLE
Rounded corner for menu bar window like in the docs

### DISCUSSION THREAD

**Discussion by damms005** [NativePHP/laravel#231] [Q&A] [answered]
Please how do I achieve this kind of rounded corner as shown in the docs Menu Bar page? Or is it a macOS thing? (I'm on Linux)

![image](https://github.com/NativePHP/laravel/assets/9839355/aced05b7-ecb8-4e68-85ca-419abe665a3e)

**Comment by martinpl** [comment, marked as answer]
You need to set MenuBar as transparent and then round content via CSS. 
```
       MenuBar::create()
            ->transparent();
```
